### PR TITLE
simplifies contributins by fully automating the dev setup. 

### PR DESCRIPTION
**Pull request by nisarhassan12**
Hi! :slightly_smiling_face:

This Pr simplifies contributions by fully automating the dev setup with Gitpod (A free online VS Code like IDE) with a single click it will launch a ready to code workspace with all the dependencies pre-installed & web server running so that anyone can straight away without wasting any time on the setup.

This is what it looks like! You can give it a try it my fork of the repo via the link below:

https://gitpod.io#https://github.com/nisarhassan12/awesome-uses/blob/master/scripts/readme-template.md

image

### CODE REVIEW

**Comment by HugoDF** [comment]
Not sure we need this, PRs get deployed to Netlify so even editing on GitHub you get feedback within a minute or so.

File: readme.md [hunk 1]
```diff
@@ -1,34 +1,42 @@
...
-███    ███    ▄█    ███   ███    ███    ▄█    ███
-████████▀   ▄████████▀    ██████████  ▄████████▀
-
-```
-
-# Awesome Uses ![Awesome][awesome-badge]
-
+# → Visit [uses.tech](https://uses.tech) for a good time
+
+[![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/wesbos/awesome-uses/blob/master/src/data.js)
```
Comment (HugoDF) [inline readme.md]: regardless of whether or not we want to integrate GitPod, this is the wrong file (it's auto-generated on pushes to master).

You want to edit ./scripts/readme-template.md instead

**Comment by nisarhassan12** [comment]
> Not sure we need this, PRs get deployed to Netlify so even editing on GitHub you get feedback within a minute or so.
> 
> Not closing the PR though, so other contributors can weigh in.

I agree! but this will make the process way easier for those interested in contributing to the Website i.e they will get everything set up with a single click like the repo cloned, dependencies pre-installed & the webserver running.

**Comment by sam-cross** [comment]
Please don't commit the readme file, edit the template in `\scripts\readme_template.md` instead

**Comment by wesbos** [comment]
Thanks a ton for this, but github already lets users edit the file directly from github. So - I don't think we need to add any more complexity to this project

**Pull request closed without merging**